Cirrious/Cirrious.MvvmCross.Touch/Views/MvxTouchViewsContainer.cs

```diff
@@ -39,6 +41,14 @@ public virtual IMvxTouchView CreateView(MvxViewModelRequest request)
 
         protected virtual IMvxTouchView CreateViewOfType(Type viewType, MvxViewModelRequest request)
         {
+            var storyboardAttribute = viewType.GetCustomAttribute<MvxFromStoryboardAttribute>();
+            if (storyboardAttribute != null) 
+            {
+                var storyboard = UIStoryboard.FromName(storyboardAttribute.StoryboardName ?? viewType.Name, null);
+                var viewController = storyboard.InstantiateViewController(viewType.Name);
+                return (IMvxTouchView) viewController;
```

Comment:
The problem is that when a custom exception handler is registered (e.g. HockeyApp, TestFlight) line 47 crashes the app if the storyboard is missing; try/catch doesn't help. In other words, the storyboard MUST exist in order for the app not to crash. I will add a try/catch anyway though, for those who are not using custom exception handlers.